modules/rules_java/5.1.0/source.json

```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "sha256-2XSi1uGlNIVtG2CtahXlfzlw2Flvuwuxe57ibKIJMyo=",
```

Comment:
```suggestion
    "integrity": "sha256-HwYZvoWmioavEJNEnpo3P9EStrzK7qGN/WoYsbv5ukA=",
```

Reply:
hmm, indeed... maybe the URL given in the WORKSPACE snippet https://github.com/bazelbuild/rules_java/releases/tag/5.1.0 doesn't actually have the same hash as the archive/refs/tags one

Reply:
Yes, those are different. The artifact under `releases` has been created manually.